docs/network-flow-visibility.md

```diff
@@ -630,6 +630,37 @@ a new manifest:
 make manifest
 ```
 
+##### Clickhouse Performance Configuration
```

Comment:
Clickhouse -> ClickHouse
same for below

Reply:
Thanks. Done